packages/xsnap/src/avaHandler.js

```diff
@@ -77,7 +77,16 @@ function handler(rawMessage) {
       const virtualObjectGlobals =
         // @ts-ignore
         // eslint-disable-next-line no-undef
-        typeof makeKind !== 'undefined' ? { makeKind, makeWeakStore } : {};
+        typeof makeKind !== 'undefined'
+          ? {
+              // @ts-ignore TODO Figure out why these are newly needed
```

Comment:
For typescript errors, I think xsnap needs a globals.d.ts file. For eslint errors, I think adding globals in a comment at the top of the file fixes it. No idea why this might change from master though

Reply:
That fixed it, thanks. Resolving.